src/main/java/me/badbones69/crazyauctions/controllers/GUI.java

```diff
@@ -302,10 +359,22 @@ public static void openBuying(Player player, String ID) {
             }
         }
         ItemStack item = data.getItemStack("Items." + ID + ".Item");
+
         List<String> lore = new ArrayList<>();
+
         for (String l : config.getStringList("Settings.GUISettings.SellingItemLore")) {
-            lore.add(l.replace("%Price%", Methods.getPrice(ID, false)).replace("%price%", Methods.getPrice(ID, false)).replace("%Seller%", data.getString("Items." + ID + ".Seller")).replace("%seller%", data.getString("Items." + ID + ".Seller")).replace("%Time%", Methods.convertToTime(data.getLong("Items." + l + ".Time-Till-Expire"))).replace("%time%", Methods.convertToTime(data.getLong("Items." + l + ".Time-Till-Expire"))));
+            DynamicReplacements dr = new DynamicReplacements(new HashMap() {{
+                put("%Price%", Methods.getPrice(ID, false));
+                put("%price%", Methods.getPrice(ID, false));
+                put("%Seller%", data.getString("Items." + ID + ".Seller"));
+                put("%seller%", data.getString("Items." + ID + ".Seller"));
+                put("%Time%", Methods.convertToTime(data.getLong("Items." + l + ".Time-Till-Expire")));
+                put("%time%", Methods.convertToTime(data.getLong("Items." + l + ".Time-Till-Expire")));
```

Comment:
This looks like a bug, where `%time%` needs to be `"Items." + ID + ".Time-Till-Expire")` but I didn't want to change any of the code itself.